src/components/computing-status/computing-type.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 export enum ComputingType {
-    LOADFLOW = 'LOADFLOW',
+    LOAD_FLOW = 'LOAD_FLOW',
```

Comment:
Why this change?

Reply:
to be coherent with the computing Status enum values